README.md

```diff
@@ -1,2 +1,48 @@
 # SimBrain
-A SimGrid-based simulator of protein spread in the brain
+A SimGrid-based simulator of protein spread in the brain.
+
+**Simulation of misfolded protein propagation in the brain as a distributed computing system**
+
+We model the brain as a distributed system where computing nodes represent brain regions
+and network links transport misfolded proteins (MPs). Nodes may produce, clear, and store MPs
+using similar processes as described in (Iturria-Medina et al. 2014). We leverage the SimGrid
+(Casanova et al. 2013) toolkit, the most advanced simulator for distributed systems, to simulate
+such a system. We compare our discrete-event simulation to the simulator in (Iturria-Medina et
+al. 2014).
+
+## Prerequisites
+
+* [SimGrid 3.17](http://simgrid.gforge.inria.fr)
+* C
+
+## Instalation
+
+* Under Debian, Ubuntu and similar systems, you can simply get the latest version of SimGrid from your distribution. For example, simply type the following in a terminal (as root):
+
+  `apt-get install simgrid`
+
+* You could also use the SimGrid docker container and get your code into the container, by typing the following in a terminal:
+
+  `sudo docker run --rm -it -v /path/to/local/directory:/SimBrain path/to/container/directory /bin/bash`
+
+  * example:`sudo docker run --rm -it -v /home/masoud/Desktop/SimBrain:/SimBrain bigdatalabteam/simgriddocker /bin/bash`
+
+## Usage
+
+Type the following in a terminal:
+
+1. List files (ls)
+2. Change directory to (cd) SimBrain
+3. List files. Then, you will see the following files inside of it: `simbrain.c  simbrain.xml  simbrain_d.xml`
```

Comment:
Step 1, 2 and 3 are not needed. Just say, `in the SimBrain directory, compile the code as follows:`